css/style.css

```
@@ -390,3 +390,98 @@
   margin-top: 4.5rem;
   color: #fff;
 }
+
+.reservation {
+  background-image: linear-gradient(
+      -13deg,
+      rgb(12, 52, 53, 0%),
+      rgb(255, 255, 255, 100%)
+    ),
+    url(/img/pattern-ellipse.png);
+  box-shadow: inset 0 0 25px rgba(1, 1, 1, 0.05);
+  display: flex;
+  gap: 3rem;
+  padding: 10rem 21.4rem 10rem 21.4rem;
+}
+
+.reservation-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 10rem;
+}
+
+.reservation-heading {
+  color: #cc9900;
+}
+
+.reservation-text {
+  text-align: center;
+  margin-top: 2.1rem;
+  width: 41rem;
+  color: #4c4c4c;
+}
+
+.reservation-form {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 3rem;
+  row-gap: 3rem;
```

It can be simplified to
```suggestion
  gap: 3rem;
```